docs/quick.dox

```diff
@@ -149,8 +149,8 @@ if (!window)
 }
 @endcode
 
-The window handle is passed to all window related functions and is provided to
-along to all window related callbacks, so they can tell which window received
+The window handle is passed to all window related functions and is provided
+along all window related callbacks, so they can tell which window received
```

Comment:
Agree.

Reply:
On second thought, why not just have:
_The window handle is passed to all window related functions and callbacks, so they can tell which window received_

Reply:
To be honest, I don't quite understand what this sentense is trying to convey. Can you please explain the concepts behind it, and we can later compact?

Reply:
My understanding; There are 2 types of window-related functions:
1. `glfwSetWindowSize(window, width, height)` **accepts** a window to operate on.
2. `glfwSetWindowSizeCallback(window, callback)` **registers** a callback, which **receives** a window, when something interesting happened to it.

Does this seem ok?

Reply:
How's that, as an RC?

_There are 3 types of window-accepting functions:_
_1. Retrieve its attributes._
_2. Mutate/Operate on it._
_3. Register a (window in question)-receiving callback, which runs when something interesting happened to it._

Reply:
Don't wanna bikeshed, but:
1. window-**accepting** or window-**related**?
2. I would suggest keeping out of the docs the callbacks themselves. For example, the function `size_callback(window, width, height)` is actually not present in the GLFW API, even though its signature is: `nameholder(window, width, height)`. It would help here to think of callbacks as **lambdas**. By giving name to those lambdas, users might confuse them with public API. Callbacks are user-defined.
3. With your suggestion, there are actually 4 types of window-related functions now (not 3), you forgot about attribute retrieving/querying like `glfwGetWindowSize(window, &width, &height)`. But again, as explained in 2. I would advise against the last bullet, bringing them back down to 3.
4. It's more generic to describe the function types by their common behavior, and not by sample functions in them, as it's easier for future versions of GLFW to change signatures than change group behaviours. That is, I would suggest only mentioning actual functions after describing their type, and making clear the fact that they represent a sample.

> that are called when an event happens and receives the window it happened to.

Even though true, it's a little misleading. User callbacks are not a hearing-all mechanism, they will not accept **any** window for which something interesting happened. Window callbacks are registered for a **given** window, and will **only** run when something interesting happens to **that and only that** window. The window they receive is the **registered window**, and it serves as an alias, when abstract callbacks are used, if not for **convenience**. Even though in some scenarios crucial, the callback receiving window might be misleading as the **any** window stated above.

Please let me know if I'm missing something.